e2e-tests/README.md

```diff
@@ -1,30 +1,22 @@
 ## Summary
 
-This is a helper library for Sidecar to run runtime tests against specific chains, at certain blocks. 
+This is a helper library for Sidecar to run e2e tests against specific chains, at certain blocks. 
 
 ## Testing
 
-The below instructions are specific to runtime-tests against one chain. 
-If you are looking to run the runtime-tests against all 3 chains (Polkadot, Kusama, Westend) then run `yarn test:init-runtime-tests` in 
+The below instructions are specific to running the e2e-tests against one chain. 
+If you are looking to run the e2e-tests against all chains (Polkadot, Kusama, Westend, Statemine) then run `yarn test:init-e2e-tests` in 
 the root directory of sidecar.
 
 ### Polkadot 
 
-Lets first get sidecar ready in a seperate terminal.
+To run the tests against a single chain, you may use the following below. For more examples, reference the `<ROOT>/package.json`
 
-```
-$ cd substrate-api-sidecar
-$ git checkout <your_branch>
-$ export SAS_SUBSTRATE_WS_URL=<network archive node>
```

Comment:
I think setting up environment variable is required otherwise it will search on local machine. So better to mention those 2 points clearly in the readme.

Reply:
So this is not required by the user because the script itself handles it. There is a [setWsUrl](https://github.com/paritytech/substrate-api-sidecar/blob/master/scripts/sidecarScriptApi.ts#L10) which the scriptsApi provides. The script itself uses a config from [config.ts](https://github.com/paritytech/substrate-api-sidecar/blob/master/scripts/config.ts#L27) in order to get the wsUrl. 

The whole idea being the user should only ever have to use `yarn test:init-e2e-tests` or for a specific chain `yarn test:init-e2e-tests:polkadot`

Reply:
That being said, I think it will be useful to add a section that gives steps to users adding their own chains to the process.